apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/BigtableVersionTest.cs

```diff
@@ -39,28 +39,30 @@ public void Comparisons()
             {
                 new BigtableVersion(6),
                 new BigtableVersion(-1),
-                new BigtableVersion(long.MaxValue)
+                new BigtableVersion(long.MaxValue / 1000)
             };
             ComparisonTester.AssertCompare(control, less, equal, greater);
             ComparisonTester.AssertCompareOperators(control, less, equal, greater);
         }
 
         [Fact]
-        public void MicrosScaleFromDateTime()
+        public void ScaleFromDateTime()
         {
             var utcNow = DateTime.UtcNow;
             var v1 = new BigtableVersion(utcNow);
             var v2 = new BigtableVersion(utcNow + TimeSpan.FromSeconds(1));
             var diff = v2.Value - v1.Value;
+            Assert.Equal(1_000, diff);
+            diff = v2.Micros - v1.Micros;
```

Comment:
I think this chunk of code would be clearer as

```csharp
Assert.Equal(1_000, v2.Value - v1.Value);
Assert.Equal(1_000_000, v2.Micros - v1.Micros);
```

Reply:
Done